src/features/content-answers/Answer.tsx

```diff
@@ -7,10 +7,17 @@ import './Answer.scss';
 
 type Props = {
     answer?: string;
+    handleScrollToBottom: Function;
     isLoading: boolean;
 };
 
-const Answer = ({ answer, isLoading }: Props) => {
+const Answer = ({ answer, handleScrollToBottom, isLoading }: Props) => {
+    useEffect(() => {
+        if (answer) {
+            handleScrollToBottom('smooth');
+        }
+    }, [answer, handleScrollToBottom]);
```

Comment:
I feel like `handleScrollToBottom` should not be in the dependency array?

Reply:
There is a rule in react that we need to include every variables and functions used inside of useEffect to be included inside of the dependency array. And here to prevent infinitely re-rendering of calling handleScrollToBottom function, we used useCallback to keep the identity same in each render